app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -20,9 +20,7 @@
       <!-- grant details row ( image + raised, address, in round, matchin, add to cart button ) -->
       <GrantDetailsRow
         :grant="grant"
-        :logoURI="
-          grantMetadata?.logoURI ? grantMetadata?.logoURI.replace('https://ipfs-dev', 'https://ipfs') : undefined
-        "
+        :logoPtr="grantMetadata?.logoPtr ? grantMetadata.logoPtr : undefined"
```

Comment:
Isn't this the same as:

```suggestion
        :logoPtr="grantMetadata?.logoPtr"
```

Reply:
Yep it is, just applied your suggestion. This is an artifact of my first refactor "design" where the code was:

```
:logoURI="grantMetadata?.logoPtr ? ptrToURI(grantMetadata.logoPtr) : undefined"
```

But then I changed the prop to `logoPtr` and forgot to change the conditional. So good catch!